examples/all-clusters-app/all-clusters-common/all-clusters-app.matter

```diff
@@ -3237,6 +3237,195 @@ server cluster ActivatedCarbonFilterMonitoring = 114 {
   command ResetCondition(): DefaultSuccess = 0;
 }
 
+/** This cluster provides an interface to the functionality of Smart Energy Demand Response and Load Control. */
+provisional server cluster DemandResponseLoadControl = 150 {
+  enum CriticalityLevelEnum : enum8 {
+    kUnknown = 0;
+    kGreen = 1;
+    kLevel1 = 2;
+    kLevel2 = 3;
+    kLevel3 = 4;
+    kLevel4 = 5;
+    kLevel5 = 6;
+    kEmergency = 7;
+    kPlannedOutage = 8;
+    kServiceDisconnect = 9;
+  }
+
+  enum HeatingSourceEnum : enum8 {
+    kAny = 0;
+    kElectric = 1;
+    kNonElectric = 2;
+  }
+
+  enum LoadControlEventChangeSourceEnum : enum8 {
+    kAutomatic = 0;
+    kUserAction = 1;
+  }
+
+  enum LoadControlEventStatusEnum : enum8 {
+    kUnknown = 0;
+    kReceived = 1;
+    kInProgress = 2;
+    kCompleted = 3;
+    kOptedOut = 4;
+    kOptedIn = 5;
+    kCanceled = 6;
+    kSuperseded = 7;
+    kPartialOptedOut = 8;
+    kPartialOptedIn = 9;
+    kNoParticipation = 10;
+    kUnavailable = 11;
+    kFailed = 12;
+  }
+
+  bitmap CancelControlBitmap : bitmap16 {
+    kRandomEnd = 0x1;
+  }
+
+  bitmap DeviceClassBitmap : bitmap32 {
```

Comment:
sadly the specs doesn't exposed these properly
this bitmap lgtm

Reply:
Please see PR description: this is targeting an outstanding spec PR, not the already-merged spec.